proto/worker.proto

```diff
@@ -202,3 +204,43 @@ message DebugStateReply {
 message PurgeTasksRequest {
     BigInt dealID = 1;
 }
+
+message WorkerMetricsRequest {}
+
+message GPUMetrics {
+    // pci bus id
```

Comment:
Please stick to go comments style, like "Id represents the PCI bus ID". Here and below.